firebase/test/core/data/repositories/stored-collections-repository.spec.ts

```diff
@@ -10,30 +10,28 @@ import createSinonStub from "#test/sinon-stub";
 describe("StoredCollectionsRepository", () => {
   let sandbox: sinon.SinonSandbox;
   let firestoreStub: sinon.SinonStubbedInstance<FirestoreGateway>;
+  let schemaStub: sinon.SinonStubbedInstance<SchemaValidator>;
   let storedCollectionsRepo: StoredCollectionsRepository;
   let transactionSpy: sinon.SinonSpy;
 
-  beforeEach(() => {
+  before(() => {
     sandbox = sinon.createSandbox();
 
-    const firestoreStubInstance = createSinonStub(FirestoreGateway, sandbox);
-    firestoreStub = firestoreStubInstance;
-
-    const schemaStubInstance = createSinonStub(SchemaValidator, sandbox);
+    const firestoreStubInstance = (firestoreStub = createSinonStub(FirestoreGateway, sandbox));
+    const schemaStubInstance = (schemaStub = createSinonStub(SchemaValidator, sandbox));
```

Comment:
I don't think moving multiple assignments to the same line is a good code style/format